net-im/spectrum2/files/spectrum2.initd

```diff
@@ -0,0 +1,40 @@
+#!/sbin/openrc-run
```

Comment:
Please modernize this:
1. Use `command` etc. to avoid redundant phases.
2. Use `checkpath` to create directories.

Reply:
Done.

Reply:
i'm not seeing the first change though.

Reply:
Because I misunderstood this. I see now, whats meant by `command`.
I've updated the script, but there is one thing, I miss. I can't set the args based on start or stop.
So I had to keep the stop function. Is there a better way?